pkg/machine/ignition/ignition.go

```diff
@@ -239,6 +241,23 @@ func (ign *DynamicIgnition) GenerateIgnitionConfig() error {
 		ignSystemd.Units = append(ignSystemd.Units, qemuUnit)
 	}
 
+	// Only AppleHv with Apple Silicon can use Rosetta
+	if ign.VMType == define.AppleHvVirt && runtime.GOARCH == "arm64" {
+		rosettaUnit := Systemd{
+			Units: []Unit{
+				{
+					Enabled: BoolToPtr(true),
+					Name:    "rosetta-activation.service",
+				},
+				{
+					Enabled: BoolToPtr(true),
+					Name:    "unregister-handler.service",
+				},
```

Comment:
just so we do not forget, if we go a ahead with https://github.com/containers/podman-machine-os/pull/8 this must be removed here

Reply:
Fixed it, and latest machine-mac CI passed !